src/virtualenv/discovery/windows/__init__.py

```diff
@@ -19,15 +18,10 @@ def propose_interpreters(spec, cache_dir, env):
     )
 
     for name, major, minor, arch, exe, _ in existing:
-        # pre-filter
-        if name in ("PythonCore", "ContinuumAnalytics"):
```

Comment:
For these two can we still match without the from exe and only for other do what you're doing now?

Reply:
Yeah we could. I was wondering whether it's worth it to make this exceptional logic for these 2 orgs? What would the additional complexity achieve?

Reply:
It is worth it. They are the most popular and will be used 99% of the time.

Reply:
OK. So the logic would be:

1. if name in ("PythonCore", "ContinuumAnalytics"), implementation becomes "CPython"
2. if implementation == "CPython", apply registry-based pre-filtering
3. in other cases, do not apply pre-filtering at all

Reply:
Yes.